CourseManager/Utilities/MainColors.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  MainColors.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-23.
+//
+
+import SwiftUI
+
+
+enum MainColors  {
```

Comment:
I think it is better to follow ideas from SwiftUI. The framework name instance of colors in singular (`Color`). So it is better to name either `AppColor` or `MainColor`